### DISCUSSION TITLE
2.1.4 Released!

### DISCUSSION THREAD

**Discussion by cmmarslender** [Chia-Network/chia-blockchain#17281] [Announcements]
### Fixed
* Update chia_rs to 0.2.15 for AMD K10 architecture (fixes #16386)

### Changed
* improved CPU usage due to tight loop in `send_transaction()`
* improve performance of `total_mempool_fees()` and `total_mempool_cost()`
* reduced the default maximum peer count to 40 from 80 (only applies to new configs)
* changed to `normal` SQlite db sync option (previously was `full`)
* reduced the mempool size to 10 blocks from 50 blocks (improves performance)
* improve performance of the mempool by batch fetching items from the db

<hr /><em>This discussion was created from the release <a href='https://github.com/Chia-Network/chia-blockchain/releases/tag/2.1.4'>2.1.4</a>.</em>

**Comment by Notargets** [comment]
I just updated to 2.1.4 from 2.1.3 after posting this thread on Full Node performance issues here: https://chiaforum.com/t/full-node-system-load-observations-questions/21604/1

It seems like you've definitely fixed the performance (regressions?) in this release - that's the good news!

From a different perspective - this single threaded server loop seems pretty overloaded - not sure what the ramifications are of hitting the CPU at 100% all the time limits are in the future, but I can say it's been unpleasant this time - not to mention making the idea of using a many-core low-single-core-perf CPU like ARM64 on RPI impractical.

Any thoughts on how hard it would be to multi-process this event loop? Sounds like there's a transactional element to it, so maybe you're using the SQLLite locking to manage your concurrency? This could be reworked...

**Reply by BrandtH22** [reply]
Hey @Notargets , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

As for the CPU consistently being at 100% that is not normal, while many optimizations to the code are underway there are many community and staff members who actively farm from rpis without issue. 

When farming compressed plots it is recommended to validate how many plots and at what compression level the hardware will support, our documents relay that without a GPU or extremely high end CPU it is not advised to farm higher than bladebit c5 compression.

For the sql thoughts, feel free to open an enhancement request outlining what you think should be changed (https://github.com/Chia-Network/chia-blockchain/discussions/new?category=ideas) or we always welcome community PR submissions 👍 

If you want to jump into our discord server or open a support discussion here on github we will be happy to help further:

Discord will be the more timely and thorough means of getting support: https://discord.gg/chia
Github support discussions are not checked as frequently but offer another means to acquire support: https://github.com/Chia-Network/chia-blockchain/discussions/new?category=support

**Comment by dodonation** [comment]
Hi since the new update since the Chias have been published, in the Gui in the history since you can find more now https://xchscan.com/ can I see the Coins habe.

I am afraid that the update will not be installed until it is not yet announced. Do you have a solution?

Thanks
